sim/hunter/serpent_sting.go

```diff
@@ -52,7 +52,7 @@ func (hunter *Hunter) getSerpentStingConfig(rank int) core.SpellConfig {
 			BonusCoefficient: spellCoeff,
 
 			OnSnapshot: func(sim *core.Simulation, target *core.Unit, dot *core.Dot, isRollover bool) {
-				damage := baseDamage + (hunter.SerpentStingAPCoeff*dot.Spell.RangedAttackPower(target))/5
+				damage := baseDamage + (hunter.SerpentStingAPCoeff*dot.Spell.RangedAttackPower(target, true))/5
```

Comment:
Could you leave a comment on these for any future person to know? Just something like "Serpent sting's bonus RAP ignores target modifiers like mob type AP and Hunter's Mark"